route53_crossfeed_app.tf

```diff
@@ -35,9 +35,9 @@ resource "aws_route53_record" "crossfeed_prod_AAAA" {
 resource "aws_route53_record" "crossfeed_prod_acm_CNAME" {
   provider = aws.route53resourcechange
 
-  name    = "_972ea3f08cb5c1e588ee885ef10d0f9e.api.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
```

Comment:
Is this change correct?  Just double-checking because this is the only meaningful change I see here - all the others are just zone ID and TTL changes.

Also, the `_972ea3...` value is the one currently live in Route53 in the COOL DNS account.

Reply:
Yep, I double-checked my references - the original commit had two entries for the `_972ea3...` value, which should only be the `api` url and there should be a different entry for the non-`api` url, but instead I'd put in two entries for `api`.

Reply:
OK, great.